pkgs/development/python-modules/fitbit/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, dateutil
+, requests_oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "fitbit";
+  version = "0.3.0";
+
+  propagatedBuildInputs = [ dateutil requests_oauthlib ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9a88a790659c318762d1a30ad3d831393044f3040ca5d24959ca14eb3c1e256a";
+  };
+
+  # Tests require "Sphinx<1.4,>=1.2".
```

Comment:
Please remove the version pinning using `substituteInPlace` in `postPatch`.

Reply:
Done. Some more changes were required to make tests actually work (turns out the source package in PyPi is missing the unit tests modules...).